### PR TITLE
[Fix] Bug fixes for the Exposure Submission Flow

### PR DESCRIPTION
**Pull request by johannesrohwer**
This PR includes multiple bug fixes and small updates for the exposure submission flow.

## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

### CODE REVIEW

**Review by inf2381 — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -435,9 +434,9 @@ Sie können Ihre Einwilligung jederzeit widerrufen, indem Sie die App löschen.
 
 "ExposureSubmissionError_RegTokenNotExist" = "Es konnte keine Übermittlungs-TAN erstellt werden. Bitte kontaktieren Sie die technische Hotline über App-Informationen -> Technische Hotline.";
 
-"ExposureSubmissionError_other" = "Es ist ein Verbindungsfehler aufgetreten ";
+"ExposureSubmissionError_other" = "Es ist ein Verbindungsfehler aufgetreten. Fehlercode: ";
 
-"ExposureSubmissionError_otherend" = "Bitte versuchen Sie es erneut.";
+"ExposureSubmissionError_otherend" = "\nBitte versuchen Sie es erneut.";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:439]: Why the newline character?

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift [hunk 2]
```diff
@@ -131,9 +131,9 @@ class DynamicTableViewImageCardCell: UITableViewCell {
 		chevron.centerYAnchor.constraint(equalTo: title.centerYAnchor).isActive = true
 
 		body.leadingAnchor.constraint(equalTo: insetView.leadingAnchor, constant: 16).isActive = true
-		body.trailingAnchor.constraint(equalTo: cellImage.leadingAnchor, constant: -16).isActive = true
+		body.trailingAnchor.constraint(equalTo: cellImage.leadingAnchor).isActive = true
 		body.topAnchor.constraint(equalTo: title.bottomAnchor, constant: 21).isActive = true
-		body.bottomAnchor.constraint(equalTo: insetView.bottomAnchor, constant: -16).isActive = true
+		body.bottomAnchor.constraint(lessThanOrEqualTo: insetView.bottomAnchor, constant: -16).isActive = true
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift:136]: Can be dangerous and result in undefined layout. Is this intended?

**Review by i336616 — APPROVED** [review]
👍

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewStepCell.swift [hunk 3]
```diff
@@ -142,18 +144,29 @@ class DynamicTableViewStepCell: UITableViewCell {
...
-
 		body.leadingAnchor.constraint(equalTo: cellIcon.trailingAnchor, constant: 10).isActive = true
 		body.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -20).isActive = true
 
+		if reducedSpacing {
+			body.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -8).isActive = true
+		} else {
+			body.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -35).isActive = true
+		}
+
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewStepCell.swift:155]: See above.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -213,9 +213,9 @@
 
 "ExposureSubmission_DataPrivacyDescription" = "Durch Antippen von „Akzeptieren“ willigen Sie ein, dass die App den Status Ihres Corona-Virus-Tests abfragen und in der App anzeigen darf. Diese Funktion steht Ihnen zur Verfügung, wenn Sie einen QR-Code erhalten und eingewilligt haben, dass Ihr Testergebnis an das Serversystem der App übermittelt werden darf. Sobald das Testlabor Ihr Testergebnis auf dem Server hinterlegt hat, können Sie das Ergebnis in der App sehen. Falls Sie Mitteilungen aktiviert haben, werden Sie auch außerhalb der App über den Eingang des Testergebnis informiert. Das Testergebnis selbst wird aus Datenschutzgründen jedoch nur in der App angezeigt. Sie können diese Einwilligung jederzeit widerrufen, indem Sie die Testregistrierung in der App löschen. Durch den Widerruf der Einwilligung wird die Rechtmäßigkeit der bis zum Widerruf erfolgten Verarbeitung nicht berührt. Weitere Informationen finden Sie unter dem Menüpunkt „Datenschutzinformation“.";
 
-"ExposureSubmissionDataPrivacy_AcceptTitle" = "Erlauben";
+"ExposureSubmissionDataPrivacy_AcceptTitle" = "Akzeptieren";
```
Comment (ChristopherSchmitz) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:216]: for consistancy reasons. stay with Erlauben and change the DPP text to "Durch Antippen von "Erlauben" willigen Sie"

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (thomasaugsten) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:439]: There is a errocode between

**Pull request merged**